text_paddle.py

```diff
@@ -60,11 +60,14 @@ def _get_input_data(self, docs, parameters):
 
         # traverse thought all documents which have to be processed
         flat_docs = []
-        for trav_path in trav_paths:
-            flat_docs.extend(docs.traverse_flat(trav_path))
+        if isinstance(trav_paths, str):
```

Comment:
This is actually handled by traverse logic itself. You do not have to do it here